resources/guides/laptop_guide.md

```diff
@@ -0,0 +1,72 @@
+
+Many Computer Science classes (COMP-1400 and COMP-1410, for example) will require programming environments that are harder to set up in Windows than in other UNIX/Linux-based OS. Although substitutes like WSL and virtual machines can help, they may not be fully compatible or require a lot of computing power. Should you choose Windows as your primary OS, you should consider setting up an SSH connection to the UWindsor's remote server on editors like Visual Studio Code or via other means. In addition, when you are working with other developers on a project, you may also have to pay attention to your line breaks (CRLF vs. LF).
+
+If you are taking classes offered by other faculties or departments (e.g. Engineering), you may have to install software that are only available on Windows. However, this usually is not a concern since such instances are quite rare, and the University of Windsor has many other computer resources to help you out if something like this happens.
+
+### Linux
+
+Linux/UNIX is very good for programming/developing because of how easy it is to set up the necessary environment. Much like MacOS, it can be difficult to game on. Depending on how important gaming is to you, you may need to use Wine, set up virtual machine, or dual-boot with Windows. Additionally, since many of the University's software (Office 365, OneDrive, Teams, etc.) are on the Microsoft platform, there _may_ be some compatibility issues.
+
+Popular Linux distros include Ubuntu, Debian, Fedora, Pop!\_OS, Linux Mint, and Arch Linux. If you are very new to Linux, then Ubuntu and Debian may be a good starting point for you simply because there is a large community using them and most likely have answers for any problem you encounter. The University, for example, uses Debian on its remote servers. If you have time and are willing to experiment, you should test out the different Linux OS since the desktop environment on Ubuntu and Debian (GNOME) can be slightly laggy on laptops without high specifications.
+
+Although Linux OS may be confusing for new users, becoming comfortable with a UNIX/Linux environment is very beneficial for you. The University generally expects you to produce codes/projects that will work on its Linux server. As well, Linux forces you to interact with the command line more, which is great if you want to learn more about how your computer works underneath.
+
+Almost all Linux distros are free, especially the popular ones. As well, people have said that Linux takes up less storage and is more battery-efficient compared to Windows. Thus, the right Linux distro will significantly optimize your laptop, especially if you are on a budget.
```

Comment:
```suggestion
Almost all Linux distros are free, especially the popular ones. In general, Linux distros require less disk space and perform better on low-end devices, compared to Windows. This is great for your budget, and also for battery life. Thus, the right Linux distro will significantly optimize your laptop, especially if you are on a budget.
```

Reply:
linux is not always better for battery life. Its actually general consensus that its worse than windows.